content/en/docs/admin/collectors.md

```diff
@@ -68,4 +63,11 @@ RT Collector enables Taranis AI to collect data from a user-defined RT instance.
   * **RT_TOKEN**: User token for the RT instance.
 
 * Optional fields:
-  * **TLP_LEVEL**
\ No newline at end of file
+  * **TLP_LEVEL**
+
+## Digest Splitting
+Digest Splitting is a feature that allows the user to split all available URLs in the located element into individual News Items.
+The Digest Splitting Limit is the maximum number of URLs that will be split into individual News Items. If the limit is reached, the remaining URLs are dropped.
+The Digest Splitting Limit is set to 30 by default but can be adjusted by the administrator. It is generally recommended to keep the limit low to avoid overloading the system, which could lead to unsuccessful gathering.
+
+_Note: Digest splitting can fail if the web page contains JavaScript. As of now, there is no workaround but it might be improved in the future releases._
```

Comment:
Simple Web Collector will fail if the web page content is only available with JavaScript. Switching to the Selenium based "Web Collector" will allow parsing JavaScript based websites, but as of right now "Web Collector" does not support Digest Splitting


Remove the _Note: at the bottom.